neuromation/cli/job.py

```diff
@@ -365,14 +367,19 @@ def job() -> None:
 @command()
 @click.argument("job")
 @click.argument("local_remote_port", type=LOCAL_REMOTE_PORT, nargs=-1, required=True)
+@click.option("--reconnect/--no-reconnect", default=True, is_flag=True, help="TODO")
```

Comment:
I think `is_flag=True` is redundant if we use "--foo/--no-foo".

Reply:
No, i have checked now :(